packages/block-library/src/image/image.js

```diff
@@ -1043,10 +1061,43 @@ export default function Image( {
 		);
 	}
 
+	/**
+	 * Set the post's featured image with the current image.
+	 */
+	const setPostFeatureImage = () => {
+		setFeaturedImage( id );
```

Comment:
```suggestion
		setFeaturedImage( id );
		createSuccessNotice( __( 'Post featured image updated.' ), {
			type: 'snackbar',
		} );
```

It would be good to give a notice here, the text may have room for improvement.

![f64055e3940f0ae33c64b4dff0b7fb57](https://github.com/user-attachments/assets/d3a74b8a-3ea0-41ee-9e4f-9690cd15609f)